### PR TITLE
feat:SS:PASinfoatTableAllOrders

### PR DESCRIPTION
**Pull request by sansito84**
**Add PAS column**
Captura de pantalla de 2023-08-24 10 27 35

**Add PAS Modal**
Captura de pantalla de 2023-08-24 10 27 44

### CODE REVIEW

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx [hunk 1]
```diff
@@ -4,17 +4,22 @@ import "./table.css";
...
     const [showModal, setShowModal] = useState(false);
+    const [modalDataPas, setModalDataPas] = useState(null);
+    const [showModalPas, setShowModalPas] = useState(false);
     const [currentPage, setCurrentPage] = useState(1);
     const [cuantityPage, setCuantityPage] = useState(0);
+    const [pasInfo, setPasInfo] = useState();
     const resultsPerPage = 7;
     const userId =
-        user.type === "superadmin" ? "" : user.type === "admin" ? "" : user.id;
+        user.type === "superadmin" ? "" : user.type === "admin" ? "" : user.id_user;
```
Comment (Patriciopg02) [inline src/components/DashBoard/TableOrders/TableAllOrders.jsx:22]:
```suggestion
        user.type !== "superadmin" && user.type !== "admin" ? user.id_user : null;
```
i think you can reduce this ternary like this.

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx [hunk 2]
```diff
@@ -81,11 +94,22 @@ const OrdersTableAll = () => {
             });
     };
 
+    const getPas = (idPas) => {
+        axios
+            .get(`${process.env.REACT_APP_URI_API}/user/getPasInfo/${idPas}`)
+            .then((response) => {
+                setPasInfo(response.data[0]);
+                setModalDataPas(response.data[0]); // Configurar modalDataPas aquí
```
Comment (Patriciopg02) [inline src/components/DashBoard/TableOrders/TableAllOrders.jsx:102]:
```suggestion
                setModalDataPas(response.data[0]); // ToDo: configure modelDataPas
```

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx (hunk 1 above)
Comment (sansito84) [reply src/components/DashBoard/TableOrders/TableAllOrders.jsx:22]: This ternary return null and need ""

Comment (Patriciopg02) [reply src/components/DashBoard/TableOrders/TableAllOrders.jsx:22]: then return "", its ok

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx [hunk 3]
```diff
@@ -178,7 +211,8 @@ const OrdersTableAll = () => {
                                             </td>
                                         )}
                                         <td data-label="Cotizado">
-                                            {user.type === "admin" || user.type === "superadmin" ? (
+                                            {user.type === "admin" ||
+                                            user.type === "superadmin" ? (
                                                 order?.cotizated === null ? (
```
Comment (Patriciopg02) [inline src/components/DashBoard/TableOrders/TableAllOrders.jsx:216]: i think this is a bit repetitive, maybe you can create a constant with this ternary like this:
`const userIsAdmin = user.type === "admin" || user.type === "superadmin";`
So the result is: 
`
                                                {userIsAdmin ? ( ...
`

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx [hunk 4]
```diff
@@ -242,6 +276,28 @@ const OrdersTableAll = () => {
                                                 </b>
                                             )}
                                         </td>
+                                        {user.type === "admin" ||
+                                        user.type === "superadmin" ? (
```
Comment (Patriciopg02) [inline src/components/DashBoard/TableOrders/TableAllOrders.jsx:280]: same this

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx (hunk 3 above)
Comment (Patriciopg02) [inline src/components/DashBoard/TableOrders/TableAllOrders.jsx:216]: i think this is a bit repetitive, maybe you can create a constant with this ternary like this:
const userIsAdmin = user.type === "admin" || user.type === "superadmin";
So the result is:
{userIsAdmin ? ( ...

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx [hunk 5]
```diff
@@ -178,7 +211,8 @@ const OrdersTableAll = () => {
                                             </td>
                                         )}
                                         <td data-label="Cotizado">
-                                            {user.type === "admin" || user.type === "superadmin" ? (
+                                            {user.type === "admin" ||
+                                            user.type === "superadmin" ? (
```
Comment (Patriciopg02) [inline src/components/DashBoard/TableOrders/TableAllOrders.jsx:215]: i think this is a bit repetitive, maybe you can create a constant with this ternary like this:
const userIsAdmin = user.type === "admin" || user.type === "superadmin";
So the result is:
{userIsAdmin ? ( ...

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx (hunk 4 above)
Comment (Patriciopg02) [inline src/components/DashBoard/TableOrders/TableAllOrders.jsx]: same here

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx (hunk 5 above)
Comment (sansito84) [reply src/components/DashBoard/TableOrders/TableAllOrders.jsx]: already changed

File: src/components/DashBoard/TableOrders/TableAllOrders.jsx (hunk 4 above)
Comment (sansito84) [reply src/components/DashBoard/TableOrders/TableAllOrders.jsx]: already changed

**Review by Patriciopg02 — APPROVED** [review]

**Pull request merged**